internal/common/metrics.go

```diff
@@ -84,16 +106,8 @@ func sendCollectDMetrics(metricPerMinute int, duration time.Duration) error {
 }
 
 func sendStatsdMetrics(metricPerMinute int, duration time.Duration) error {
-	// https://github.com/cactus/go-statsd-client#example
-	statsdClientConfig := &statsd.ClientConfig{
-		Address:     ":8125",
-		Prefix:      "statsd",
-		UseBuffered: true,
-		// interval to force flush buffer. full buffers will flush on their own,
-		// but for data not frequently sent, a max threshold is useful
-		FlushInterval: 300 * time.Millisecond,
-	}
-	client, err := statsd.NewClientWithConfig(statsdClientConfig)
+	// https://github.com/DataDog/datadog-go#metrics
+	client, err := statsd.New("127.0.0.1:8125", statsd.WithMaxMessagesPerPayload(100), statsd.WithNamespace("statsd"), statsd.WithoutTelemetry())
```

Comment:
`Datadog` has more flexibility than I imagine:
* It can [send events and service checks](https://github.com/DataDog/datadog-go#usage)
* It also supports [distribution](https://docs.datadoghq.com/metrics/custom_metrics/dogstatsd_metrics_submission/#distribution)
* Supports [more options in control the packsize under high throughput environment - e.g control the packet size when sending through UDP ](https://github.com/DataDog/datadog-go/blob/7d2f91bc7a91003561b6657d5974d92584833f98/statsd/options.go)